nautobot/extras/tests/example_jobs/test_atomic_transaction.py

```diff
@@ -17,7 +17,7 @@ def run(self, fail=False):
         try:
             Status.objects.create(name="Test database atomic rollback 1")
             if fail:
-                raise Exception("simulated failure")
+                raise Exception("simulated failure")  # pylint: disable=broad-exception-raised
```

Comment:
TODO: Replace these naked exceptions and pylint disables with a real exception (e.g. [`JobTaskRunFailed`](https://github.com/nautobot/nautobot/blob/feature/765-jobs-overhaul/nautobot/extras/jobs.py#L80-L81))